x-pack/plugins/ml/server/routes/schemas/results_service_schema.ts

```diff
@@ -26,7 +26,7 @@ export const anomaliesTableDataSchema = schema.object({
   dateFormatTz: schema.string(),
   maxRecords: schema.number(),
   maxExamples: schema.maybe(schema.number()),
-  influencersFilterQuery: schema.maybe(schema.any()),
+  filterQuery: schema.maybe(schema.any()),
```

Comment:
The change to the schema here is causing the anomalies table in the ML app to fail to load when a filter is applied e.g in the Anomaly Explorer:

![image](https://user-images.githubusercontent.com/7405507/155569147-fab6bef9-790c-4fef-988c-610f0a59499c.png)

There will be quite a few of the client side files that need changing to pass `filterQuery` rather than `influencersFilterQuery` to the `/api/ml/results/anomalies_table_data` route.

It may be simpler for you to back out the rename here.

Reply:
Yes, I am reverting the renaming. Thanks for catching it. 🙇